app/javascript/packages/verify-flow/index.tsx

```diff
@@ -33,6 +35,19 @@ interface VerifyFlowProps {
 }
 
 export function VerifyFlow({ initialValues = {}, basePath, appName, onComplete }: VerifyFlowProps) {
+  function trackVisitedStepEvent(stepName) {
+    if (stepName === 'personal_key') {
+      trackEvent('IdV: personal key visited');
```

Comment:
We could do it here or in a separate pull request, but I think for LG-6201 to be complete, we need to figure out how to avoid [prefixing](https://github.com/18F/identity-idp/blob/84f0706fb273a77fb140af733d8a43b34074fb0a/app/controllers/frontend_log_controller.rb#L9) for these events, since otherwise we don't have strict parity with how it was working previously.